src/module/token/token-hud.js

```diff
@@ -0,0 +1,124 @@
+export class SFRPGTokenHUD extends TokenHUD {
+
+    /**
+     * @override
+     * Calls setCondition from the SFRPG actor before calling super
+     * @param {Event} event 
+     * @param {any} param1 - contains overlay boolean
+     * @returns super
+     */
+    async _onToggleEffect(event, { overlay = false } = {}) {
+        event.preventDefault();
+        event.stopPropagation();
+
+        let img = event.currentTarget;
+        let isEnabled = $(img).hasClass('active');
+
+        if (img.dataset.statusId && this.object.actor) {
+            let conditionId = img.dataset.statusId;
+            await this.object.actor.setCondition(conditionId, !isEnabled, {overlay: overlay});
+        }
+
+        return !isEnabled;
+    }
+
+    /**
+     * @override
+     * Toggles active and overlay classes on the status images.
+     */
+    refreshStatusIcons() {
+        const effects = this.element.find(".status-effects")[0];
+        const statuses = this._getStatusEffectChoices();
+
+        for (let img of $('[src]', effects)) {
+            const status = statuses[img.getAttribute("src")] || {};
+
+            img.classList.toggle("overlay", !!status.isOverlay);
+            img.classList.toggle("active", !!status.isActive);
+        }
+    }
+
+    /**
+     * @override
+     * Localizes and sorts the effects list before calling super
+     */
+    _getStatusEffectChoices() {
+        CONFIG.statusEffects = CONFIG.statusEffects.sort(function (a, b) {
+            let aid = (a.label != undefined ? game.i18n.localize(a.label) : a.id);
```

Comment:
Localization of config elements should happen in sfrpg.js, check the hook for setup.